bot/utility/utility.py

```diff
@@ -95,6 +95,47 @@ async def about(self, ctx: commands.Context):
         embed.add_field(name="Links:", value=str_links)
         await ctx.send(embed=embed)
 
+    @commands.group(name='howto', invoke_without_command=True)
+    @command_log
+    async def howto(self, ctx: commands.Context, subcommand: Optional[str]):
+        """Handler for the `howto` command.
+
+        Explains how to do various things
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            subcommand (Optional[str]): The subcommand the user wants to invoke.
+        """
+        message = "Ich kenne diesen Befehl leider nicht. Hast du vielleicht einen der folgenden gemeint?\n" if subcommand else None
+
+        await ctx.send(message, embed=discord.Embed(
+            title="Howtos",
+            description="\n".join([c.name + ": " + c.description for c in self.howto.commands])
```

Comment:
What value would `c.description` have in this context?

If I'm reading this correctly, the list inside the embed would look something like this:
> eat: Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam.
> sleep: Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam.
> code: Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam.
> repeat: Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam.

Maybe add some sort of bullet point like a `-` in front of the individual commands and make them bold? I'm also not sure if we really need the descriptions in the help message but I guess I would first have to see embed myself for making a good decision.

Reply:
You decorate a command in a group with @howto.command(name, description).
That description is also the value of c.description
![howto](https://user-images.githubusercontent.com/75306520/128886899-ecdd96dd-7c88-4666-8dbe-8344f123cefb.png)